lib/app/routes/exercises.rb

```diff
@@ -72,6 +72,10 @@ class Exercises < Core
         flash[:success] = "#{exercise.problem.name} in #{exercise.problem.track_id} is now reactivated."
         redirect '/'
       end
+
+      get '/five_a_aday/:key' do |key|
```

Comment:
Typo on the route pattern. Let's use hyphens rather than underscores, as well: `/five-a-day/`